.github/semantic.yml

```diff
@@ -0,0 +1,28 @@
+# Config ref: https://github.com/Ezard/semantic-prs
+
+# Validate the PR title, and ignore all commit messages
+titleOnly: true
+
+# The values allowed for the "type" part of the PR title/commit message.
+# e.g. for a PR title/commit message of "feat: add some stuff", the type would be "feat"
+types:
+  - ci
+  - chore
+  - build
+  - docs
+  - feat
+  - fix
+  - perf
+  - refactor
+  - revert
+  - style
+  - test
+
+# The values allowed for the "scope" part of the PR title/commit message.
+# e.g. for a PR title/commit message of "feat(awesome-feature): add some stuff",
+#      the type would be "awesome-feature"
```

Comment:
```suggestion
#      the scope would be "awesome-feature"
```

Reply:
Good catch! (This was copy-pasted, so it probably exists in their docs also.)